src/logic/LocalMessageCache.tsx

```diff
@@ -79,12 +80,25 @@ const isValid = (m: CachedMessage): boolean => {
     );
 };
 
-const getQueue = (slotName: SlotName): CachedMessage[] => {
+const getQueue = (slotName: SlotName, errorHandler?: ErrorHandler): CachedMessage[] => {
     const queue = readQueue(slotName);
     const validQueue = queue.filter((i) => isValid(i));
     const unexpiredQueue = validQueue.filter((i) => hasNotExpired(i));
 
     if (queue.length !== unexpiredQueue.length) {
+        const expiredMessages = queue.length - unexpiredQueue.length;
```

Comment:
Thoughts on naming this `expiredMessageCount` to make it's usage slightly clearer?